lib/fastlane/plugin/nowsecure/actions/nowsecure_action.rb

```diff
@@ -72,7 +74,13 @@ def self.available_options
             optional: false,
             verify_block: proc do |value|
               UI.user_error!("Could not find a file at '#{value}'") unless File.exist?(value)
-            end)
+            end),
+          FastlaneCore::ConfigItem.new(key: :nowsecure_endpoint,
+            env_name: "NOWSECURE_ENDPOINT",
+            description: "NowSecure Tenant Endpoint",
+            optional: false,
```

Comment:
Let's mark this as optional.
```suggestion
            optional: true,
```